src/assets/scripts/client/info/AirportInfoController.js

```diff
@@ -0,0 +1,284 @@
+import $ from 'jquery';
+import { leftPad } from '../utilities/generalUtilities';
+import { radiansToDegrees } from '../utilities/unitConverters';
+import AirportController from '../airport/AirportController';
+import SimClockController from './SimClockController';
+import EventBus from '../lib/EventBus';
+import { INVALID_NUMBER } from '../constants/globalConstants';
+import { EVENT } from '../constants/eventNames';
+import { AIRPORT_INFO_TEMPLATE } from './airportInfoTemplate';
+import { PERFORMANCE } from '../constants/aircraftConstants';
+
+/**
+ * @property INFO_VIEW_SELECTORS
+ * @type {object<string, string>}
+ * @final
+ */
+const INFO_VIEW_SELECTORS = {
+    CLOCK_LABEL: '.js-airportInfo-clock-label',
+    CLOCK_VALUE: '.js-airportInfo-clock-value',
+    WIND_LABEL: '.js-airportInfo-wind-label',
+    WIND_VALUE: '.js-airportInfo-wind-value',
+    ALTIMETER_LABEL: '.js-airportInfo-altimeter-label',
+    ALTIMETER_VALUE: '.js-airportInfo-altimeter-value',
+    ELEVATION_LABEL: '.js-airportInfo-elevation-label',
+    ELEVATION_VALUE: '.js-airportInfo-elevation-value'
+};
+
+/**
+ * Gets information about the current airport, specifically
+ * the airport's elevation, wind speed and direction, and altimeter.
+ *
+ * @class AirportInfoController
+ */
+export default class AirportInfoController {
```

Comment:
Is this file supposed to be included in this PR?

Reply:
no